android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -169,6 +171,12 @@ public void onUpdate() {
       }
     }
 
+    // orientation
+    if (!ScreenFragment.checkIfChildWithConfig(getScreen() != null ? getScreen().getChildAt(1) : null)) {
```

Comment:
This look weird. What's the point of calling checkIfChildWithConfig(null)? Does it have side effect? I also commented that the name of this method isn't really very informative. Also its role isn't very clear by looking at that name neither by looking at the way we use it in different places.